BUILD_DETAIL.md

```diff
@@ -66,6 +66,22 @@ $ bundle exec cucumber
 $ bin/cucumber
 ```
 
+## Snippets
+
+RSpec Rails uses snippets, self-contained examples that are used to cover
+cases and regressions that don't need a full-blown example application to
+reproduce.
+
+Snippets reuse the already installed gems, and don't attempt to install gem
+versions that are not on the system already to prevent version mismatches.
+
+Run with:
+
+```
+$ cd snippets
+$ find . -name '*.rb' | xargs -I_ -L1 sh -c "echo Running _ && ruby _"
```

Comment:
Do you want to extract that to a `script/run_snippets`?

Reply:
Good idea. I've seen CI fail with a nonsense message about `_` in this statement, so using a `for` inside a single runnable script file is totally reasonable.
Done, and squashed.